osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/mbus/CoupleMbusDeviceCommandExecutorTest.java

```diff
@@ -35,6 +34,10 @@
 @ExtendWith(MockitoExtension.class)
 public class CoupleMbusDeviceCommandExecutorTest {
 
+  @InjectMocks
+  private final CoupleMBusDeviceCommandExecutor commandExecutor =
+      new CoupleMBusDeviceCommandExecutor();
```

Comment:
When using InjectMocks, new CoupleMBusDeviceCommandExecutor() assignment is not required